docs/configuration.md

```diff
@@ -99,7 +99,7 @@ Using type `etcd` will make k0s to create and manage an elastic etcd cluster wit
 
 ### `spec.network`
 
-- `provider`: Network provider, either `calico` or `custom`. In case of `custom` user can push any network provider.
+- `provider`: Network provider, either `calico`, `kuberouter` or `custom`. In case of `custom` user can push any network provider. (default `calico`)
```

Comment:
default shouldn't be cube-router?

Reply:
👍  good catch, default indeed after this PR is merged is kuberouter. I'll fix